src/query/api/v1/handler/placement/get_test.go

```diff
@@ -113,6 +113,5 @@ func TestPlacementGetHandler(t *testing.T) {
 
 	resp = w.Result()
 	body, _ = ioutil.ReadAll(resp.Body)
```

Comment:
I think metalint is complaining about this useless line now that we're not using it.